pdns/recursordist/meson_options.txt

```
@@ -23,3 +23,4 @@ option('snmp', type: 'boolean', value: false, description: 'Enable SNMP')
 option('dnstap', type: 'feature', value: 'auto', description: 'Enable DNSTAP support through libfstrm')
 option('libcurl', type: 'feature', value: 'auto', description: 'Enable Curl support')
 option('nod', type: 'boolean', value: true, description: 'Enable Newly Obserbed Domains')
```

Ok it's not part of this PR but at some point, try and smuggle this
```suggestion
option('nod', type: 'boolean', value: true, description: 'Enable Newly Observed Domains')
```